make dist failure

Trying to build source package for debian, I've encountered this problem on develop branch : if you try `make dist`, here is the result:

    make  dist-bzip2 am__post_remove_distdir='@:'
    make[1]: Entering directory '/root/greg/gregorio-dev'
    if test -d "gregorio-4.0.0-rc1"; then find "gregorio-4.0.0-rc1" -type d ! -perm -200 -exec chmod u+w {} ';' && rm -rf "gregorio-4.0.0-rc1" || { sleep 5 && rm -rf "gregorio-4.0.0-rc1"; }; else :; fi
    test -d "gregorio-4.0.0-rc1" || mkdir "gregorio-4.0.0-rc1"
    (cd src && make  top_distdir=../gregorio-4.0.0-rc1 distdir=../gregorio-4.0.0-rc1/src \
     am__remove_distdir=: am__skip_length_check=: am__skip_mode_fix=: distdir)
    make[2]: Entering directory '/root/greg/gregorio-dev/src'
    make[2]: *** No rule to make target 'dump/dump.h', needed by 'distdir'. Arrêt.
    make[2]: Leaving directory '/root/greg/gregorio-dev/src'
    Makefile:455: recipe for target 'distdir' failed
    make[1]: *** [distdir] Error 1
    make[1]: Leaving directory '/root/greg/gregorio-dev'
    Makefile:553: recipe for target 'dist' failed
    make: *** [dist] Error 2

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -21,7 +21,7 @@ AM_CFLAGS =
 bin_PROGRAMS = gregorio
 gregorio_SOURCES = characters.c gregorio-utils.c messages.c struct.c \
 				   unicode.c struct.h messages.h unicode.h characters.h \
-				   plugins.h config.h dump/dump.c dump/dump.h \
+				   plugins.h config.h dump/dump.c \
 				   gregoriotex/gregoriotex-write.c gregoriotex/gregoriotex.h
 
 @MK@ifneq ($(wildcard ../.git),)

